### ISSUE TITLE
Error From Application

### ISSUE THREAD

**Issue by TechGirl99** [basildane/WakeOnLAN#85]
![wakeonlan](https://user-images.githubusercontent.com/32715170/31456525-ee266872-ae6e-11e7-956e-56daf3d78e1b.jpg)

I just opened the application and this happened.

**Comment by TechGirl99** [comment]
Also forgot to note, I ran an update yesterday to the software, so it is the latest version.

**Comment by basildane** [comment]
Please install and run the debug version of the program.
[checked build](https://cloud.aquilatech.com/s/N7s38McTaCJNa1x)

Please install and run this.  Do your “search for machines”.  When the search is completed, close the program.

You will find a file in your “My Documents” folder called “wakeonlan.tracelog”.
Please send me that file.  I’m hoping it will give us a hint on where to look for the problem.
You can upload the log to here [upload log files](https://cloud.aquilatech.com/s/cKHVPHPXfaYdbjn)

Thank you!

**Comment by TechGirl99** [comment]
I have completed your request.  Please let me know if you need anything else.

Kind regards,

Tammy



Tammy L. Nice



Managed Services











E: tnice@empyrion.net<mailto:tnice@empyrion.net>

[cid:image002.jpg@01D2E13E.2807D6B0]



P: 1.604.746.3797 – Abbotsford



P: 1.604.824.3777 – Chilliwack



F: 1.888.697.0576



W: www.empyrion.net<http://www.empyrion.net/>

**Comment by basildane** [comment]
Yes, reading it.  Thank you.
I'm not seeing what is causing that error yet, and I have not been able to reproduce it.  Any thoughts?
Have you used WOL previously?  When did the error start appearing?  Or has it always been their?

Windows 10?
Are you a local administrator on this machine?

**Comment by TechGirl99** [comment]
It is on a Server and we have been using it for about 5 months with no issues.  Yesterday I was prompted for an upgrade and I installed it.  This morning a computer didn’t turn on so I opened the program and I got all those boxes.

Should I uninstall and reinstall?





Tammy L. Nice



Managed Services











E: tnice@empyrion.net<mailto:tnice@empyrion.net>

[cid:image002.jpg@01D2E13E.2807D6B0]



P: 1.604.746.3797 – Abbotsford



P: 1.604.824.3777 – Chilliwack



F: 1.888.697.0576



W: www.empyrion.net<http://www.empyrion.net/>

**Comment by basildane** [comment]
Did the problem just appear when you open the task scheduler?  Or do you execute some function?
I'm trying to replicate the problem.

**Comment by TechGirl99** [comment]
Hi,

The first time it was when I removed a device.  The second time it was when I was searching to add new devices.

I am going for lunch will be back in 1 hour if you have any more questions.



Tammy L. Nice



Managed Services











E: tnice@empyrion.net<mailto:tnice@empyrion.net>

[cid:image002.jpg@01D2E13E.2807D6B0]



P: 1.604.746.3797 – Abbotsford



P: 1.604.824.3777 – Chilliwack



F: 1.888.697.0576



W: www.empyrion.net<http://www.empyrion.net/>

**Comment by TechGirl99** [comment]
Hi, I uninstalled and reinstalled the previous version we had installed.  It is working again.  I will not apply the update.

**Comment by basildane** [comment]
I have been working on a "checked build" debug version for you, do you not want to fix it?
I would have had it out to you earlier but my last 24 hours have been indescribable..   

It's nearly ready.  I would like to find the bug and fix it, with your help.

**Comment by TechGirl99** [comment]
I would like to help you, however this client is very demanding.  He wants it working and doesn’t want to wait.  He expects his machines to be shut down at the end of the day and wol at 6:00 am.  If that doesn’t happen he’s on the phone wanting to know what happened.  He won’t pay for my time to fix a broken software and my boss won’t pay me either.

I had to reinstall because when I opened it I got a message that the program was being used by trace log and wouldn’t open.  So I decided to try to older version as I know it worked.

Where you able to recreate the issue?  I can install an upgrade and let you know if I have any issues.

**Comment by basildane** [comment]
Do to other emergencies I have not been able to focus on this, sorry about that.
I built a debug build that has additional debug info in it.
That build is here. [WakeOnLAN_2.11.19.20](https://cloud.aquilatech.com/s/y8wNrqXrxYFpZtB)

After running that, it will create My Documents\wakeonlan.tracelog.
If you could upload that to [uploads](https://cloud.aquilatech.com/s/cKHVPHPXfaYdbjn) it would be great.

Also, if you could upload your machines.xml that would be extremely helpful.
I would really appreciate at least the xml file.  I really want to fix this properly.
Thanks for you patience.

**Comment by TechGirl99** [comment]
Okay I have a pretty busy week, but I will help you out.  I can do this on my lunch hour.  Today I have to do some errands, but I can do it tomorrow.





Tammy L. Nice



Managed Services











Email: tnice@empyrion.net<mailto:tnice@empyrion.net>

[cid:image002.jpg@01D2E13E.2807D6B0]



Phone: 1.604.746.3797 – Abbotsford Office



Phone: 1.604.824.3777 – Chilliwack Office



Fax: 1.888.697.0576



Web: www.empyrion.net

**Comment by basildane** [comment]
Thanks.  No rush.

**Comment by basildane** [comment]
I'm assuming this is fixed in 2.11.20.

**Comment by TechGirl99** [comment]
Hello,

I don’t really know if it was fixed, we reverted back to the previous version and did not upgrade.  We will not be upgrading the product as the client will not pay for troubleshooting when the previous version works just fine.  So there may or may not be an issue with the upgrade, but my boss will not pay me to spend time on troubleshooting either.   Unless someone else is having the same issue, I would consider this closed as well.

Kind regards,

Tammy




Tammy L. Nice



Managed Services











Email: tnice@empyrion.net<mailto:tnice@empyrion.net>

[cid:image002.jpg@01D2E13E.2807D6B0]



Phone: 1.604.746.3797 – Abbotsford Office



Phone: 1.604.824.3777 – Chilliwack Office



Fax: 1.888.697.0576



Web: www.empyrion.net
